include/onnxruntime/core/optimizer/graph_transformer_utils.h

```diff
@@ -4,24 +4,31 @@
 #pragma once
 
 #include "core/optimizer/graph_transformer.h"
+#include "core/optimizer/rule_based_graph_transformer.h"
 #include "core/optimizer/rewrite_rule.h"
 
 namespace onnxruntime {
 
 namespace transformer_utils {
 
-/* Generates all predefined rules for this level
-*  If rules_to_enable is not empty returns intersection of predefined rules and rules_to_enable 
-*/
-std::vector<std::unique_ptr<RewriteRule>> GenerateRewriteRules(const TransformerLevel& level, 
+/** Generates all predefined rules for this level.
+   If rules_to_enable is not empty, it returns the intersection of predefined rules and rules_to_enable. */
+std::vector<std::unique_ptr<RewriteRule>> GenerateRewriteRules(const TransformerLevel& level,
```

Comment:
I was planning to make this change but since you are already touching this - please change const TransformerLevel& to TansformerLevel.